### DISCUSSION TITLE
Do not implement Stable Diffusion 3!

### DISCUSSION THREAD

**Discussion by motolo** [Acly/krita-ai-diffusion#801] [General]
Given the situation with stable diffusion 3, I recommend not implementing it, it would be wasted resources.  Rather, I believe that the team should focus on animations, even short ones but with better ease of use.

Thanks for the splendid work

**Comment by NoMansPC** [comment]
I disagree. It's mostly because I don't use Stable Diffusion in another UI, but I think this plug-in should still support SD3. It already supports SD 1.5 and XL.

**Reply by shagratt** [reply]
I agree with motolo. It should support it! but not right now. You can get better results with sdxl and adding detail loras,etc. this situation will change with time of course.  Animation and other features will add more value at this moment

**Comment by motolo** [comment]
After the latest tests, the exit from the STable diffusion team of Comfyanonimous, after the Civitai ban, after revealing that sd3 2b is a failed training and not having released the 4b model, I would say that there are the conditions to say that SD3 must remain out of the project...
